platform/vs/docker-ptf-sai.mk

```
@@ -0,0 +1,9 @@
+# docker image for docker-ptf-sai
+
+DOCKER_PTF_SAI = docker-ptf-sai.gz
+DOCKER_PTF_BASE = docker-ptf.gz
+$(DOCKER_PTF_SAI)_PATH = $(DOCKERS_PATH)/docker-ptf-sai
+$(DOCKER_PTF_SAI)_DEPENDS += $(LIBTHRIFT_2) $(PYTHON3_THRIFT_2)
```

Should be updated
```suggestion
$(DOCKER_PTF_SAI)_DEPENDS += $(LIBTHRIFT_0_14_1) $(PYTHON3_THRIFT_0_14_1)
```

Also, I believe you should add `$(PYTHON_SAITHRIFT)` and `$(PTF)`